libraries/AP_HAL_ChibiOS/hwdef/SPEDIXF405/README.md

```diff
@@ -0,0 +1,99 @@
+# SPEDIX F405 Flight Controller
+
+The SPEDIX F405 is a flight controller based on the STM32F405 MCU.
+
+## Features
+
+* MCU - STM32F405 32-bit processor running at 168 MHz
+* IMU - ICM42688
+* Barometer - SPL06
+* OSD - AT7456E
+* Onboard Flash: 4MByte
+* 6x UARTs
+* 9x PWM Outputs (8 Motor Output, 1 LED)
+* Battery input voltage: 2S-6S
+* BEC 3.3V 0.5A
+* BEC 5V 3A
+* BEC 9V 3A for video
+
+## Pinout
+
+![SPEDIX F405 Board](SpedixF405.png "SPEDIX F405")
+
+## UART Mapping
+
+* SERIAL0 -> USB
+* SERIAL1 -> UART1 (MSP DisplayPort, DMA-enabled)
+* SERIAL2 -> UART2 (Spare)
+* SERIAL3 -> UART3 (RX, DMA-enabled)
+* SERIAL4 -> UART4 (VTX)
+* SERIAL5 -> UART5 (ESC Telemetry)
+* SERIAL6 -> UART6 (GPS)
+
+## RC Input
+
+RC input is configured by default via the USART2 RX input. It supports all serial RC protocols except PPM.
+
+Note: If the receiver is FPort the receiver must be tied to the USART2 TX pin , RSSI_TYPE set to 3,
+and SERIAL2_OPTIONS must be set to 7 (invert TX/RX, half duplex). For full duplex like CRSF/ELRS use both
+RX2 and TX2 and set RSSI_TYPE also to 3.
+
+## FrSky Telemetry
+ 
+FrSky Telemetry is supported using an unused UART, such as the T3 pin (UART3 transmit).
+You need to set the following parameters to enable support for FrSky S.PORT:
+ 
+  - SERIAL3_PROTOCOL 10
+  - SERIAL3_OPTIONS 7
+
+## OSD Support
+
+OSD is supported via the AT7456E chip connected to SPI2.
```

Comment:
replace with standard:
Onboard OSD using OSD_TYPE 1 (MAX7456 driver) is supported by default. Simultaneously, DisplayPort OSD is available on the HD VTX connector.

Reply:
Fixed